src/main/java/com/github/dockerjava/core/DockerClientConfig.java

```diff
@@ -418,8 +418,8 @@ public final DockerClientConfigBuilder withDockerConfig(String dockerConfig) {
         }
 
         public final DockerClientConfigBuilder withDockerTlsVerify(String dockerTlsVerify) {
-            this.dockerTlsVerify = BooleanUtils.toBoolean(dockerTlsVerify.trim())
-                    || BooleanUtils.toBoolean(dockerTlsVerify.trim(), "1", "0");
+            String trimmed = dockerTlsVerify.trim();
+            this.dockerTlsVerify = "true".equals(trimmed) || "1".equals(trimmed);
```

Comment:
LOL, There always two ways. 1. make maximum use of Utils; 2. make code as simple as possible. 

But I'm the latter one, I perfect has third libraries as little as possible.

As the former one, if BooleanUtils has a method to pass default value, it will looks very good for eyes, just like:

`toBoolean(str, trueStr, falseStr, defaultValue);`

Or if docker-java does not want user to set invalid value to DOCKER_TLS_VERIFY, it should throw exception directly.